lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -525,9 +525,9 @@ void OpenCLFunction::executeConvolution(const OCLConvolutionInst *CC) {
 
   // The global work for the N dimension (which covers all pixels) should be big
   // enough to cover all outputs for the correctness.
-  if (global[0] * fw_div_N < odim.h * odim.w) {
-    global[0] = (odim.h * odim.w - 1) / fw_div_N + 1;
-  }
+  //  if (global[0] * fw_div_N < odim.h * odim.w) {
+  //  global[0] = (odim.h * odim.w - 1) / fw_div_N + 1;
+  //}
```

Comment:
Can you add this fix in a separate PR or at least in a separate commit? It is not related to group convolutions at all.

Reply:
OK. Will revert it in this PR and create a new one.